src/UXClient/Components/GeoProcessGraphic/GeoProcessGraphic.ts

```diff
@@ -0,0 +1,179 @@
+      }
+    });
+    this.map.setCamera({
+        center: this.center,
+        bearing: this.bearing,
+        pitch: this.pitch,
+        zoom: this.zoom,
+        maxZoom: this.maxZoom,
+        minZoom: this.minZoom,
+        type: "fly",
+        duration: this.duration
+    });
+    return Promise.resolve({
+        graphic: this.map,
+        width: this.width,
+        height: this.height
+      });
+  }
+  
+  protected getDataPoints(promise: Array<IGeoProcessGraphicLabelInfo>){
+    let dataPoints = promise.map((r): IGeoProcessGraphicLabelInfo => {
+        return this.parseTsqResponse(r);  
+      });
+      this.updateDataMarkers(dataPoints);
+  }
+
+  protected parseTsqResponse(response) {
+    let parsedResponse = {};
+    if (response && response.properties){
+      for (let i =0; i<response.properties.length; i++){
+          response.properties[i] && response.properties[i].name && response.properties[i].values ?
+            parsedResponse[response.properties[i].name] = response.properties[i].values[0]
+              : null;
+      }
+    }
+    return parsedResponse;
+  }
+
+  protected updateDataMarkers(dataPoints: Array<any>) {
+      for(let i = 0; i < dataPoints.length; i++) {
+        let lat = dataPoints[i][this.tsqExpressions[i].positionXVariableName];
+        let lon = dataPoints[i][this.tsqExpressions[i].positionYVariableName];
+        this.map.markers.getMarkers()[i].setOptions({
+            position: [lat, lon]
+        });
+        let dataPointArr = Object.entries(dataPoints[i]);
+        
+        this.map.popups.getPopups()[i].setOptions({
+            position: [lat, lon],
+            content: this.createTable(dataPointArr, i)
+        });
+        console.log(this.map.popups.getPopups()[i]);
+      }
+    }
+
+  protected createTable(dataPointArr, idx){
+    let gpgTooltipDiv = document.createElement('div');
+    gpgTooltipDiv.className = 'tsi-gpgTooltip tsi-'+ this.theme;
+
+    let gpgTooltipInnerDiv = document.createElement('div');
+    gpgTooltipInnerDiv.className = 'tsi-gpgTooltipInner';
+
+    let gpgTooltipTitleDiv = document.createElement('div');
+    gpgTooltipTitleDiv.className = 'tsi-gpgTooltipTitle';
+
+    let title = document.createTextNode(this.tsqExpressions[idx].alias);
+    gpgTooltipTitleDiv.appendChild(title);
+
+    let gpgTooltipTable =  document.createElement('table');
+    gpgTooltipTable.className = 'tsi-gpgTooltipValues';
+    gpgTooltipTable.classList.add('tsi-gpgTooltipTable');
+
+    for (let i = 0; i < dataPointArr.length; i++){
+      let spacer = document.createElement('tr');
+      spacer.className = 'tsi-gpgTableSpacer';
+      gpgTooltipTable.appendChild(spacer);
+
+      let gpgTooltipValueRow = document.createElement('tr');
+
+      let gpgValueLabelCell = document.createElement('td');
+      gpgValueLabelCell .className = 'tsi-gpgValueLabel';
+      let labelName = document.createTextNode(dataPointArr[i][0]);
+      gpgValueLabelCell .appendChild(labelName);
```

Comment:
```suggestion
      gpgValueLabelCell.appendChild(labelName);
```

nit, spaces before method calls here